Sourcery/Models/Enum.swift

```diff
@@ -51,6 +51,7 @@ class Enum: Type {
     internal(set) var rawType: String? {
         didSet {
             if let rawType = rawType {
+                hasRawType = true
```

Comment:
Not exactly, as we can extend enum what can add inherited types, but they can not be raw types in this case.